setup.py

```diff
@@ -0,0 +1,32 @@
+from setuptools import (setup, find_packages)
+from trs_filer import __version__
+
+with open('README.md', 'r') as fh:
+    long_description = fh.read()
+
+setup(
+    name='trs-filer',
+    version=__version__,
+    author='ELIXIR Cloud & AAI',
+    author_email='nagorikushagra9@gmail.com',
+    description='Lightweight, flexible Flask/Gunicorn-based \
+                    GA4GH TRS implementation',
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    license='Apache License 2.0',
+    url='https://github.com/elixir-cloud-aai/trs-filer.git',
+    packages=find_packages(),
+    keywords=(
+        'ga4gh trs elixir rest restful api app server openapi '
+        'swagger mongodb python flask'
+    ),
+    classifiers=[
+        'License :: OSI Approved :: Apache Software License',
+        'Development Status :: 3 - Alpha',
+        'Intended Audience :: Science/Research',
+        'Topic :: Scientific/Engineering :: Bio-Informatics',
+        'Natural Language :: English',
+        'Programming Language :: Python :: 3.6',
```

Comment:
Well, we are testing against `3.7` and `3.8`, so I think you can include these as well - Python 3.6 is pretty old...